forbidden-imports.toml

```diff
@@ -153,6 +153,9 @@ hint = 'upgrade to V8'
 [forbidden."Nri.Ui.Select.V7"]
 hint = 'upgrade to V8'
 
+[forbidden."Nri.Ui.Select.V8"]
+hint = 'upgrade to V9'
```

Comment:
I can't find anything in the readme about how this file works, but should all the forbidden versions of `Nri.Ui.Select` now hint 'upgrade to V9' ?